modules/_node-scripts/cssLoad/writeCSSLoaderJavaScriptModules.mjs

```diff
@@ -0,0 +1,37 @@
+import fs from 'fs/promises';
+import path from 'path';
+
+import getFlatName from '../util/getFlatName.mjs';
+
+export default async function writeCSSLoaderJavaScriptModules(projectExports, projectWebContextPath) {
```

Comment:
Is this only ever used for exported dependencies of css? If so, can we name it more explicitly? Something like `writeCSSLoaderForExports`

Reply:
How about `writeCSSExportsLoaderModules`? Loaders alone doesn't convey that they are in fact ES modules... :thinking:

Reply:
done